lnrpc/invoicesrpc/addinvoice_test.go

```diff
@@ -667,76 +673,28 @@ var populateHopHintsTestCases = []struct {
 }, {
 	name: "populate hop hints stops after having considered all the open " +
 		"channels",
-	setupMock: func(h *hopHintsConfigMock) {
-		fundingOutpoint1 := wire.OutPoint{Index: 9}
-		chanID1 := lnwire.NewChanIDFromOutPoint(&fundingOutpoint1)
-		remoteBalance1 := lnwire.MilliSatoshi(10_000_000)
-
-		fundingOutpoint2 := wire.OutPoint{Index: 2}
-		chanID2 := lnwire.NewChanIDFromOutPoint(&fundingOutpoint2)
-		remoteBalance2 := lnwire.MilliSatoshi(1_000_000)
-
-		allChannels := []*channeldb.OpenChannel{
-			// After sorting we will first process chanID1 and then
-			// chanID2.
+	setupMock:   setupMock2Channels,
+	maxHopHints: 10,
+	amount:      100_000_000,
+	expectedHopHints: [][]zpay32.HopHint{
```

Comment:
The test refactoring, can that go into a separate commit? That way the actual fix and functional change is observable in isolation.